src/vorta/views/misc_tab.py

```diff
@@ -78,16 +86,51 @@ def populate(self):
 
                 # create widget
                 cb = QCheckBox(translate('settings', setting.label))
-                cb.setCheckState(setting.value)
+                cb.setChecked(setting.value)
```

Comment:
I didn't get this actually 😅 , the previous one could have worked as well?

Reply:
I don't know why `setCheckState` worked before. But it did not in my testing. The docs confirm that `setChecked` is the right way to go since it expects a boolean value while a CheckState is one of 0, 1 or 2.